packages/hardhat-core/src/internal/hardhat-network/stack-traces/error-inferrer.ts

```diff
@@ -1442,6 +1514,11 @@ export class ErrorInferrer {
 
     return this._failsRightAfterCall(trace, callStepIndex);
   }
+
+  private _isPanicReturnData(returnData: Buffer): boolean {
```

Comment:
This feels a bit duplicated. Can't you do `new ReturnData(returnData).is...`?

Reply:
Yes, good catch.